SU2_CFD/src/solver_direct_mean.cpp

```diff
@@ -16675,16 +16677,14 @@ void CNSSolver::BC_ConjugateHeat_Interface(CGeometry *geometry, CSolver **solver
       /*--- Compute the normal gradient in temperature using Twall ---*/
 
       There = nodes->GetTemperature(Point_Normal);
-      Tconjugate = GetConjugateHeatVariable(val_marker, iVertex, 0);
-
-      HF_FactorHere = thermal_conductivity*config->GetViscosity_Ref()/dist_ij;
-      HF_FactorConjugate = GetConjugateHeatVariable(val_marker, iVertex, 2);
+      Tconjugate = GetConjugateHeatVariable(val_marker, iVertex, 0)/Temperature_Ref;
 
-      Twall = (There*HF_FactorHere + Tconjugate*HF_FactorConjugate)/(HF_FactorHere + HF_FactorConjugate);
-
-      // this will be changed soon...
-      Twall = GetConjugateHeatVariable(val_marker, iVertex, 0);
+      // feasible alternative to compute (conjugate) wall termperature
+      // HF_FactorHere = thermal_conductivity*config->GetViscosity_Ref()/dist_ij;
+      // HF_FactorConjugate = GetConjugateHeatVariable(val_marker, iVertex, 2);
+      // Twall = (There*HF_FactorHere + Tconjugate*HF_FactorConjugate)/(HF_FactorHere + HF_FactorConjugate);
```

Comment:
If that's feasible consider adding it as an option, otherwise remove it

Reply:
Alright ;)